docs/detections/add-exceptions.asciidoc

```diff
@@ -62,7 +62,7 @@ image::images/rule-exception-tab.png[Detail of rule exceptions tab]
 . In the *Add rule exception* flyout, name the exception. 
 . Add conditions that define the exception. When the exception's query evaluates to `true`, rules don't generate alerts even when their criteria are met.
 +
-NOTE: When you create a new exception from an alert, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the **Add comments** section.
+NOTE: When you create a new exception from an alert, exception conditions are auto-populated with relevant alert data, including data from custom highlighted fields. A comment describing this action is also automatically added to the **Add comments** section.
```

Comment:
```suggestion
NOTE: When you create a new exception from an alert, exception conditions are auto-populated with relevant alert data, prioritizing data matching any custom highlighted fields. A comment describing this action is also automatically added to the **Add comments** section.
```

Reply:
Thanks for the edits, @yctercero :) I made some minor tweaks - let me know if they work:

```suggestion
NOTE: When you create a new exception from an alert, exception conditions are auto-populated with relevant alert data. Data from custom highlighted fields are listed first. A comment that describes the auto-generated exceptions is also added to the **Add comments** section.
```

Reply:
That sounds great! Thanks so much.